### PR TITLE
View edited tracks in options

### PR DESCRIPTION
**Pull request by inverse**
Still WIP but wanted some input.

refs #1348

### CODE REVIEW

File: options/options.js [hunk 1]
```diff
@@ -214,6 +215,29 @@ require([
...
+
+			const localCache = ChromeStorage.getNamespace('LocalCache');
+
+			let cache = $('<ul class="list-unstyled"></ul>');
+			localCache.get((data) => {
+				if (Object.keys(data).length === 0) {
+					cache.append($('<li>').text('No items in the cache.'));
+				} else {
+					$.each(data, function(cachedKey, cachedData) {
+						cache.append($('<li>').text(cachedData.artist + ' - ' + cachedData.track));
```
Comment (alexesprit) [inline options/options.js]: Can we use template literals in this code block?
```js
let { artist, track } = cachedData;
cache.append($('<li>').text(`${artist} - ${track});
```

**Comment by alexesprit** [comment]
What about removing items from the cache? Don't know exactly how to make this feature to look better; I suggest to add an "X" icon right after an item text.

File: options/options.js (hunk 1 above)
Comment (inverse) [inline options/options.js]: We could try it :+1:

**Comment by inverse** [comment]
We could always iterate on this :+1:

**Comment by inverse** [comment]
@alexesprit should we merge this and iterate?

**Pull request merged**